### PR TITLE
Make menu functional basic display block/none on click

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/assets/stylesheets/components/_navbar.scss [hunk 1]
```diff
@@ -9,8 +9,47 @@
   font-weight: bold;
   .content {
     position: absolute;
-    left: 50%;
-    top: 50%;
+    left: 35%;
+    top: 35%;
```
Comment (hmatringe) [inline app/assets/stylesheets/components/_navbar.scss]: Using relative values for vertical positioning is tricky in my experience. Please turn into to an absolute value. No pb using relative values for horizontal positioning though.

**Comment by ghost** [comment]
@hmatringe please review. I have fixed the bugs and modified scroll and other ui bugs that appeared due to menu.

File: app/assets/stylesheets/config/_fonts.scss [hunk 2]
```diff
@@ -13,7 +13,7 @@ $small-font-size:             80% !default;
 $font-weight-lighter:         lighter !default;
 $font-weight-light:           300 !default;
 $font-weight-normal:          400 !default;
-$font-weight-bold:            700 !default;
+$font-weight-bold:            600 !default;
```
Comment (hmatringe) [inline app/assets/stylesheets/config/_fonts.scss]: Let's just create a utility class for the missing ones (400-500)

File: app/assets/stylesheets/components/_navbar.scss (hunk 1 above)
Comment (ghost) [reply app/assets/stylesheets/components/_navbar.scss]: Done

**Pull request merged**